<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterForm</class>
 <widget class="QWidget" name="FilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox:!editable, QComboBox::drop-down:editable {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
	stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}
/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
	stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="sourceGB">
      <property name="title">
       <string>Original Model</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableView" name="sourceView"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="proxyGB">
      <property name="title">
       <string>Proxy Model</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTableView" name="proxyView"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QDateEdit" name="toDateEdit"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="filtSyntaxCB"/>
         </item>
         <item row="1" column="0">
          <widget class="QDateEdit" name="fromDateEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="filtCaseSensCB">
           <property name="text">
            <string>Case sensitive</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbFilterPattern">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="modelColumn">
            <number>3</number>
           </property>
           <property name="notnull" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="filtPattLE">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>search pattern</string>
           </property>
           <property name="notnull" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
